spec/tasks/unify_task_test.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+class UnifyTaskTest
+  describe 'unify' do
+
+    before do
+      Rake.application.rake_require 'tasks/unify'
+    end
+
+    it "should find correct duplicates" do
+      expect {
+        Rake::Task["unify"].execute
+      }.to output("Duplicate of: JUnit with id: 677333953 Duplicate: cunit with id: 625945042.\nDuplicate of: WebComponents with id: 1031312693 Duplicate: Web Components with id: 157721798.\n").to_stdout
```

Comment:
These IDs might have been static by luck, I would assume as soon as something changes in our fixtures they won't be the same.
Therefore don't use static ids, find the actual models and use their id.

Oh and I also changed your message so the skill is failing anyway :grin: